Page.html

```diff
@@ -289,21 +289,55 @@
         var form = document.getElementById('article-form');
         form.style.display = "none";
 
+        var formIsValid = true;
+
         var configDiv = document.getElementById('config');
         configDiv.style.display = "none";
 
         var loadingDiv = document.getElementById('loading');
         loadingDiv.style.display = "block";
 
-        if (formObject.submitted === "Preview") {
+        var errorMessage = "";
+        var selectedAuthors = $('#article-authors').select2('data')
+        var customByline = document.getElementById("article-custom-byline").value;
+
+        var headline = document.getElementById('article-headline');
+        var searchTitle = document.getElementById('article-search-title');
+        var searchDescription = document.getElementById('article-search-description');
+
+        if ( (selectedAuthors === undefined || selectedAuthors.length <= 0) && ( customByline === null || customByline === undefined || customByline === "") ) {
+          errorMessage += "<br>Author or custom byline is required.";
+          formIsValid = false;
+        }
+        if (headline.value === "" || headline.value === undefined || headline.value === "") {
```

Comment:
Why is `headline.value === ""` (and similar on the if conditions below these) here twice?